web/.env.example

```
@@ -0,0 +1,7 @@
+VITE_FIREBASE_API_KEY=
+VITE_FIREBASE_AUTH_DOMAIN=
+VITE_FIREBASE_PROJECT_ID=
+VITE_FIREBASE_STORAGE_BUCKET=
+VITE_FIREBASE_MESSAGING_SENDER_ID=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_FIREBASE_MESSAGING_SENDER_ID key should go before the VITE_FIREBASE_PROJECT_ID key

(UnorderedKey)